### ISSUE TITLE
Tap for Apache Phoenix

### ISSUE THREAD

**Issue by ksranga** [Cascading/cascading-jdbc#28]
Anyone know if a Tap exist for Phoenix? I see a slideshare mention that HomeAway created one but cant seem to find it - maybe its not open sourced. If one doesnt exist would this be the right place? Create a sub-project like its been done for Oracle, mySql etc?

**Comment by fs111** [comment]
Yes, HomeAway has been working on this and they had to check with their legal departement before they could contribute it. I don't know what the current status is. I will see, if check if there is any progress.

**Comment by fs111** [comment]
hi @ksranga,

the code that HomeAway developed lives here: https://github.com/homeaway/cascading-jdbc/tree/homeaway-wip-2.7

They have not made a formal contribution back to us, but maybe you can give it a try and see if you can move that forward.
